apps/emqx_rule_engine/src/emqx_rule_funcs.erl

```diff
@@ -690,6 +691,16 @@ find_s(S, P, Dir) ->
         SubStr -> SubStr
     end.
 
+jq(FilterProgram, JSONBin)
```

Comment:
I have added a `-spec` in e814b065fd95f79bf795b52cdcfa21bb3016811d. (I did not add a spec in the initial commit since there are no specs for the other functions in the file)